src/main/webapp/edit.html

```diff
@@ -0,0 +1,76 @@
+    <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.5.0/css/bootstrap.min.css" 
+      integrity="sha384-9aIt2nRpC12Uk9gS9baDl411NQApFmC26EwAOH8WgZl5MYYxFfc+NcPb1dKGj7Sk" crossorigin="anonymous">
+    <script src="https://code.jquery.com/jquery-3.3.1.slim.min.js" integrity="sha384-q8i/X+965DzO0rT7abK41JStQIAqVgRVzpbzo5smXKp4YfRvH+8abtTE1Pi6jizo" crossorigin="anonymous"></script>
+    <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
+    <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="style.css">
+    <script src="config.js"></script>
+    <script src="editScript.js"></script>
+    <script src="authScript.js"></script>
+    <script src="createNavLinks.js"></script>
+  </head>
+  <body>
+    <nav class="navbar navbar-expand-lg navbar-dark bg-dark" id="header">
+      <div id="header-text-container">
+        <a class="navbar-brand" href="/" id="navbar-title">TravIS</a>
+        <button class="navbar-toggler" type="button" data-toggle="collapse" data-target="#navbarText" aria-controls="navbarText" aria-expanded="false" aria-label="Toggle navigation">
+          <span class="navbar-toggler-icon"></span>
+        </button>
+        <div class="collapse navbar-collapse" id="navbarText">
+          <ul class="nav navbar-nav navbar-right ml-auto">
+            <li class="nav-item">
+              <a class="nav-link" href="/">Create New Trip</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" href="/trips/">Trips</a>
+            </li>
+            <li class="nav-item">
+              <a class="nav-link" id="sign-out-link">Sign out</a>
+            </li>
+          </ul>
+        </div>
+      </div>
+    </nav>
+    <div id="content">
+      <div id="button-group-nav">
+        <a id="button-group-nav-calendar">
+          <button type="button" class="btn btn-outline-primary btn-xlarge">Calendar</button>
+        </a>
+        <a id="button-group-nav-maps">
+          <button type="button" class="btn btn-outline-primary btn-xlarge">Maps</button>
+        </a>
+        <a id="button-group-nav-edit">
+          <button type="button" class="btn btn-primary btn-xlarge">Edit</button>
+        </a>
+      </div>
+      <div class="row py-3" id="edit-page-columns">
+        <div class="col-4 order-2">
+          <div class="sticky-top" id="sidebar-container"> 
+            <div class="nav flex-column" id="sidebar-bg">
+              <div class="form-group sidebar-input-form">
+                <label for="inputPoi">Add a Point of Interest</label>
```

Comment:
I'm not sure why you need all of these right now but I assume you will be using this breakup later

Reply:
Yeah, the long list of nested `div`s definitely isn't ideal, but in order to make a two-column structure with a background on the sidebar that rests _slightly below_ the top, this is the only solution I found. If it's possible to make this same code without all the nested `div`s, that'd be ideal, but for now I'm planning to leave it like this.